tests/andromede/test_ac_link.py

```diff
@@ -0,0 +1,385 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+import pytest
+
+from andromede.expression import param, var
+from andromede.expression.expression import literal, port_field
+from andromede.expression.indexing_structure import IndexingStructure
+from andromede.libs.standard import BALANCE_PORT_TYPE, DEMAND_MODEL, GENERATOR_MODEL
+from andromede.model import (
+    Constraint,
+    ModelPort,
+    PortField,
+    PortType,
+    float_parameter,
+    float_variable,
+    model,
+)
+from andromede.model.model import PortFieldDefinition, PortFieldId
+from andromede.simulation import OutputValues, TimeBlock, build_problem
+from andromede.study import (
+    ConstantData,
+    DataBase,
+    Network,
+    Node,
+    PortRef,
+    create_component,
+)
+
+"""
+Port transfers flow and voltage angle.
+"""
+AC_PORT = PortType(id="AC_LINK", fields=[PortField("angle"), PortField("flow")])
+
+"""
+Node has 2 ports: one for angle dependent connections,
+one for power-only connections.
+Should we relax constraints on ports compatibility to allow to have only one here?
+"""
+AC_NODE_MODEL = model(
```

Comment:
Move all the models to `libs` ? By gathering all models in the libs directory we will have a clear of all used models and this will ease the work to organize the models libraries

Reply:
As a first step I have added a `libs` directory in this "models" test directory, to gather tested models.

This contains the library in YAML format, instead of the previous python definitions.

It still "links" to the balance port type defined in the python "standard" library, which I did not changed in the frame of this PR. It would probably be interesting to move this library too to YAML, but I would not do it in this PR.